services/web/server/src/simcore_service_webserver/projects/_jobs_repository.py

```diff
@@ -62,12 +62,13 @@ async def list_projects_marked_as_jobs(
         user_id: UserID,
         offset: int = 0,
         limit: int = 10,
-        job_parent_resource_name_filter: str | None = None,
+        job_parent_resource_name_prefix: str | None = None,
     ) -> tuple[int, list[ProjectJobDBGet]]:
         """
         Lists projects marked as jobs for a specific user and product
-        """
 
+        Example: job_parent_resource_name = "/solvers/solver1"
```

Comment:
I still don't get how this example helps. if you have a real description on what should go there, probably better to add a link instead.